tests.cpp

```diff
@@ -279,45 +279,46 @@ TEST_CASE_METHOD(CoherentUnitTestFixture, "test_grover")
     //
     // The target probability here is looking for 100 in the low 8 bits, and
     // expecting to find 155 in the upper register.
-    const int TARGET_PROB = 100 + (155 << 8);
+    const int TARGET_PROB = 100;
 
     // Create a table with decreasing values to search through
     unsigned char toSearch[256];
 
     // Create the lookup table
-    for (i = 0; i < 256; i++) {
+    for (i = 0; i < 156; i++) {
```

Comment:
This really needs a comment for why it’s a decreasing value for the first 156 values, and an increasing value for the last set.

Reply:
I noticed this after the PR, as well. The loop has actually become superfluous. I'm removing it entirely.